android/src/main/java/com/rssignaturecapture/RSSignatureCaptureView.java

```diff
@@ -86,23 +82,26 @@ public RSSignatureCaptureView(Context context, SignatureCallback callback) {
 	}
 
 	/**
-	* Get signature
-	*
-	* @return
+	* Get signature as a Bitmap
 	*/
 	public Bitmap getSignature() {
+		// Initialize a Bitmap to contain the signature
+		Bitmap signatureBitmap = Bitmap.createBitmap(this.getWidth(), this.getHeight(), Bitmap.Config.ARGB_8888);
 
-		Bitmap signatureBitmap = null;
-
-		// set the signature bitmap
-		if (signatureBitmap == null) {
-			signatureBitmap = Bitmap.createBitmap(this.getWidth(), this.getHeight(), Bitmap.Config.RGB_565);
-		}
-
-		// important for saving signature
+		// This view's pixels constitute the signature, so draw this view onto the signatureBitmap
 		final Canvas canvas = new Canvas(signatureBitmap);
 		this.draw(canvas);
 
+		// The signatureBitmap now contains the signature with a white background,
+		// so we will change it to a transparent background.
+		for (int x = 0; x < signatureBitmap.getWidth(); x++) {
```

Comment:
there are two for loops. What about the performance of the component ? Did you do any performance test.